sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_models.py

```diff
@@ -665,28 +621,373 @@ class AccessPolicy(BlobAccessPolicy):
     :paramtype start: ~datetime.datetime or str
     """
 
-    def __init__(self, permission=None, expiry=None, **kwargs):
+    permission: Optional[Union[FileSystemSasPermissions, str]] = None  # type: ignore [assignment]
+    """The permissions associated with the shared access signature. The user is restricted to
+        operations allowed by the permissions."""
```

Comment:
You can undo this- I believe that we decided to not do this specifically for `from_string` probably because we didn't want to unnecessarily have these ignores
See:
[File-share](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py#L311)
[Blobs](https://github.com/Azure/azure-sdk-for-python/blob/697369efcaa93035b9c644b63aef612f5b30b21d/sdk/storage/azure-storage-blob/azure/storage/blob/_models.py#L1159)